src/utils/triangles-calculation.ts

```diff
@@ -0,0 +1,45 @@
+import type { MeshAttribute } from '@loaders.gl/schema';
```

Comment:
I suggest moving this file in `attributes-validation` folder.

Reply:
Done